src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -471,7 +471,7 @@ public List<string> PullFiles(string devicePath, string localPath)
             var result = allDevicesAndTheirProperties
                 .Where(kvp => !string.IsNullOrEmpty(kvp.Value) && kvp.Value.Split().Contains(apkRequiredProperty))
                 .ToDictionary(kvp => kvp.Key, kvp => kvp.Value) as Dictionary<string, string>;
-            

```

Comment:
@premun can the .editoconfig avoid this empty space changes?

Reply:
I don't think `.editorconfig` has anything to do with trimming trailing whitespace. I personally don't mind these changes. It helps though when they happen on empty lines since you won't ever git blame those